components/button.templ

```diff
@@ -7,6 +7,7 @@ const (
 	ButtonType_Secondary
 	ButtonType_Accent
 	ButtonType_Inverted
+	ButtonType_Confirm
```

Comment:
Ik zou verwachten dat je daar gewoon de primary button zou voor kunnen gebruiken. Zoals we doen bij opslaan enzo.

Reply:
Goh, ik weet niet. In dit geval gaat het echt over een Ja/Nee-antwoord. Dan lijkt groen/rood me een beter design

Reply:
Nee, Wouter heeft gelijk, zie bv ook [de stijlgids (pagina 8)](https://workdrive.zoho.eu/file/aoe5y844ef3a6d18f40ca8dab7603483c35ba)
Als er maar 2 knoppen zijn is het altijd primary en secondary waarbij de primary de affirmatieve (of meestgebruikte) optie voorstelt. Rood/groen is zelfs minder toegankelijk, de meest voorkomende vorm van kleurenblindheid is Daltonisme, waarbij rood en groen er exact hetzelfde uitzien. Vandaar dat onze primary en secondary geïnverteerde varianten zijn van elkaar, dat is voor iedere mogelijke vorm van kleurenblindheid duidelijk. We willen ook zeker geen "Confirm" button type want dat gaat mensen juist nog verder aanzetten om daar iets anders dan primary voor te gebruiken 😛

Reply:
Ok, ik pas het aan!